tests/foreman/ui/test_subscription.py

```diff
@@ -205,10 +205,14 @@ def test_positive_access_with_non_admin_user_with_manifest(
         default_organization=org,
     ).create()
     with target_sat.ui_session(test_name, user=user.login, password=user_password) as session:
-        assert (
-            session.subscription.search(f'name = "{DEFAULT_SUBSCRIPTION_NAME}"')[0]['Name']
-            == DEFAULT_SUBSCRIPTION_NAME
-        )
+        all_subscriptions = session.subscription.read_subscriptions()
+        assert len(all_subscriptions) > 0
+        for subscription in all_subscriptions:
+            if subscription['Name'] == DEFAULT_SUBSCRIPTION_NAME:
+                assert True
+                break
+        else:
+            pytest.fail("Default subsciption not found")
```

Comment:
Would this work for you @vijaysawant ?
```suggestion
        assert any([sub['Name'] == DEFAULT_SUBSCRIPTION_NAME for sub in subscription]), 'Default subsciption not found'
```

Reply:
@vsedmik well it was suggestion to me when I was running pre-commit locally and it was failing for previous changes, I will try you suggestion as well and will update code accordingly.
Thanks

Reply:
Yeah you may need to run pre-commit to fix the line length. I just typed the idea here.